pagopa/pn-bff
# Intermediari b2b - pn-12358

## Short description

Created new api for intermediari b2b

## List of changes proposed in this pull request

- Created openapi files
- Created clients, mappers, services and components
- Created tests

## How to test

Run mvnw clean install -> open postam and calls the api -> check that everything is ok
